_migration-assistant/migration-console/migration-console-commands-references.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Command reference
-nav_order: 35
+nav_order: 40
 parent: Migration console
 ---
```

Comment:
This page does not have a heading 1.

Reply:
"Help command" should be "Help"

Reply:
See line 8.

Reply:
The "Migration console" page does not have a heading 1. And the "Help command" heading in the Command reference page should be "Help" (to match other command name headings on the page).